### PR TITLE
Update messagebird to smsc-sim-bind2

### PR DESCRIPTION
**Pull request by jazin-mendoza**

### CODE REVIEW

**Comment by jazin-mendoza** [comment]
[Update messagebird to smsc-sim as its another endpoint bind](https://trello.com/c/knLQ5Ysh/344-update-messagebird-to-smsc-sim-as-its-another-endpoint-bind)

**Comment by mico-gongob-2019** [comment]
reviewing

**Comment by jazin-mendoza** [comment]
[Update messagebird to smsc-sim as its another endpoint bind](https://trello.com/c/knLQ5Ysh/344-update-messagebird-to-smsc-sim-as-its-another-endpoint-bind)

File: .env.smsc-sim-bind2 [hunk 1]
```diff
@@ -0,0 +1,5 @@
+SMSC_PORTS=34569
+
+TLV_TAG=0x1400
+COUNTRY_CODE=123
+NETWORK_CODE=456
```
Comment (mico-gongob-2019) [inline .env.smsc-sim-bind2:5]: can you change this to something slightly different from bind1's network code. `457`

**Pull request merged**